detections/endpoint/cisco_isovalent___potential_escape_to_host.yml

```
@@ -0,0 +1,87 @@
+name: Cisco Isovalent - Potential Escape to Host
+id: 2b8a7a21-bec6-4e1f-84c4-7b319f45d2ab
+version: 1
+date: '2025-11-18'
+author: Bhavin Patel, Splunk
+type: Anomaly
+data_source:
+- Cisco Isovalent Process Exec
+status: production
+description: |
+  This analytic detects potential container escape or reconnaissance attempts by monitoring for the rapid execution of multiple suspicious Linux commands (nsenter, mount, ps aux, and ls) within a short time window. The search aggregates process execution logs into 5-minute buckets and identifies when two or more distinct commands occur in quick succession. This behavior is noteworthy because attackers often chain these commands together to pivot from a container into the host, enumerate processes, or browse filesystems. For a SOC, catching these clustered command executions is important because it highlights possible adversary activity attempting to break isolation and escalate privileges inside a Kubernetes environment.
+search: |
+  `cisco_isovalent_process_exec`
+
+  (
+      process_name IN ("nsenter","mount","ps","ls")
+      OR process="*nsenter*" OR process="*mount*" OR process="*ps aux*" OR process="*ps -ef*"
```

This feels better to read.

I also wanted to point out that by saying `process_name == ps` it actually include all CLI options like `ps -ef` or `ps aux` so we actually do not need them, but I will leave them just in case there are is one but no the other.

```suggestion
      process_name IN ("nsenter","mount","ps","ls")
      OR
      process IN ("*nsenter*", "*mount*", "*ps aux*", "*ps -ef*")
```